### PR TITLE
Improved sizing BP to plan for buffer and fixed benchmark link

### PR DESCRIPTION
**Pull request by berndruecker**
## What is the purpose of the change

Improved based on learnings in the field, made recommendation to add buffer more explizit, fixed link to point to the proper benchmark project

## When should this change go live?

No dependencies

### CODE REVIEW

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 1]
```diff
@@ -124,7 +124,9 @@ Spinning up a Camunda Platform 8 Cluster means you run multiple components that
...
+Zeebe scales horizontally by adding more cluster nodes (pods). This is **limited by the [number of partitions](/docs/components/zeebe/technical-concepts/partitions/)** configured for a Zeebe cluster, as the work within one partition cannot be parallelized by design. Hence, you need to define enough partitions to utilize your hardware. The **number of partitions cannot be changed after the cluster was initially provisioned** (at least not yet), elastic scalability of partitions is (not yet) possible.
+
+If you foresee that the load will increase over time, you should prepare by configuring more partitions than you need just now, as this will be your buffer if your load increases later on. For examle, you could multiply the number of paritions you need for your current load by four to add buffer, which typically has a neglectable impact on performance.
```
Comment (christinaausley) [inline docs/components/best-practices/architecture/sizing-your-environment.md:129]: Small typos here -- would rewrite as:

**"If you anticipate the load increasing over time, prepare by configuring more partitions than you currently need as a buffer. For example, you could multiply the number of partitions you need for your current load by four to add a buffer. This typically has just a small impact on performance."**

Thoughts?

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 2]
```diff
@@ -233,4 +235,4 @@ If you are in doubt about which package to choose, you can do a load test with a
 
 This is recommended if you exceed the above numbers of three million process instances per day.
 
-You can look at the [Zeebe benchmark project](https://github.com/camunda-cloud/zeebe/blob/develop/benchmarks/setup/README.md#benchmarking-camunda-cloud-saas). While this project will not run out-of-the-box (e.g. you need need to build starter and worker code yourself and use self-created docker images), you can use it as a starting point for own endavours.
+You can look at the [Camunda 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark). While this project will not run out-of-the-box (e.g. you need need to build starter and worker code yourself and use self-created docker images), you can use it as a starting point for own endavours.
```
Comment (christinaausley) [inline docs/components/best-practices/architecture/sizing-your-environment.md:238]: To clean up a few typos, would rewrite as:

**Take a look at the [Camunda Platform 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark). While this project will not run out of the box (you need need to build starter and worker code yourself and use self-created Docker images), you can use it as a starting point for your own projects.**

Thoughts?

File: docs/components/best-practices/architecture/sizing-your-environment.md (hunk 1 above)
Comment (berndruecker) [reply docs/components/best-practices/architecture/sizing-your-environment.md:129]: Works 👍

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 3]
```diff
@@ -124,7 +124,9 @@ Spinning up a Camunda Platform 8 Cluster means you run multiple components that
 
 All components are clustered to provide high-availability, fault-tolerance and resiliency.
 
-Zeebe scales horizontally by adding more cluster nodes (pods). This is limited by the so-called partition size of a Zeebe cluster, as the work within one partition cannot be parallelized by design. Hence, you need to define enough partitions to utilize your cluster or to have some buffer if your load increases later on. The number of partitions cannot be changed after the cluster was initially provisioned (at least not yet), so elastic scalability of cluster nodes is (not yet) possible.
+Zeebe scales horizontally by adding more cluster nodes (pods). This is **limited by the [number of partitions](/docs/components/zeebe/technical-concepts/partitions/)** configured for a Zeebe cluster, as the work within one partition cannot be parallelized by design. Hence, you need to define enough partitions to utilize your hardware. The **number of partitions cannot be changed after the cluster was initially provisioned** (at least not yet), elastic scalability of partitions is (not yet) possible.
```
[the remark this answers is not in the document]
Comment (berndruecker) [reply docs/components/best-practices/architecture/sizing-your-environment.md:127]: ok

File: docs/components/best-practices/architecture/sizing-your-environment.md (hunk 2 above)
Comment (berndruecker) [reply docs/components/best-practices/architecture/sizing-your-environment.md:238]: Works - just realized it is wrong / outdated :-) Lets's simplify to the following - WDYT? 

Take a look at the [Camunda Platform 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark) as a starting point for your own benchmarks.

**Comment by berndruecker** [comment]
@christinaausley Looks all good to me - I also fixed an error I missed before. Happy if you simply merge it with those changes if OK for you?

**Review by christinaausley — APPROVED** [review]

**Pull request merged**